src/typing/__tests__/typingReducer-test.js

```diff
@@ -32,150 +32,136 @@ describe('typingReducer', () => {
       : { ...base, op: 'stop', type: EVENT_TYPING_STOP };
   };
 
-  describe('EVENT_TYPING_START', () => {
-    test('adds sender as currently typing user', () => {
-      const initialState = NULL_OBJECT;
-
-      const action = egTypingAction({
+  describe('REGISTER_COMPLETE', () => {
+    test('resets state to initial state', () => {
+      const initialState = eg.baseReduxState.typing;
+      const action1 = egTypingAction({
         op: 'start',
         sender: user1,
         recipients: [user1, eg.selfUser],
-        time: 123456789,
+        time: 123456889,
       });
+      const prevState = typingReducer(initialState, action1);
```

Comment:
This should also have a line checking `prevState` is not equal to `initialState`, right? Like with `fetchingReducer` above.